### PR TITLE
[VideoToolbox] Adds bindings from Xcode 9 Beta 1 to Stable

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/VideoToolbox/VTDefs.cs [hunk 1]
```diff
@@ -246,4 +250,28 @@ public enum VTDownsamplingMode {
 		Decimate,
 		Average
 	}
+
+	[Mac (10,13), iOS (11,0), TV (11,0)]
+	public struct VTInt32Point {
```
Comment (rolfbjarne) [inline src/VideoToolbox/VTDefs.cs]: These two structures look useless, they're not used anywhere else from what I can see. Maybe just remove them?

Comment (dalexsoto) [reply src/VideoToolbox/VTDefs.cs]: Yeah good point, they seem to be unused

**Comment by dalexsoto** [comment]
@rolfbjarne thanks, good catch!

**Comment by dalexsoto** [comment]
build

**Comment by dalexsoto** [comment]
Test failure is unrelated (bug #55719)

**Pull request merged**